pom.xml

```diff
@@ -88,9 +88,8 @@
 	</build>
 	<repositories>
 		<repository>
-			<id>eclipse-luna</id>
-			<url>http://download.eclipse.org/releases/luna/</url>
-			<layout>p2</layout>
```

Comment:
not sure what the layout entry does, but mentioning it in case it was removed inadvertently